packages/core/directives/map.ts

```diff
@@ -267,6 +267,9 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
 
   private _observableSubscriptions: Subscription[] = [];
 
+  private _mapLongitude: number = 0;
+  private _mapLatitude: number = 0;
+
```

Comment:
Use Google's LatLng type.

Reply:
Using the same type as the original inputs/properties, latitude and longitude.